homeassistant/components/switcher_kis/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
```

Comment:
Why don't you use a duration selector for the services?

Reply:
The timer service does use duration, the auto-off service is a little bit complicated, it exists since core 0.93 and it can set a time from 1 minute to 23:59 hours, which will be a little bit complicated using a duration service (and a breaking change)